### ISSUE TITLE
Problem geting JQuery Slider to work on Touchscreens

### ISSUE THREAD

**Issue by Zenid** [furf/jquery-ui-touch-punch#129]
I am trying to get my slider test page to work with touchscreen devices, I'm using a touch-punch library, and have these references for my script code behind my slider test page:

```
<link rel="stylesheet" href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css">
<script src="Scripts/jqueryv1.10.2.js" type="text/javascript"></script>
<script src="Scripts/jquery-ui.js" type="text/javascript"></script>
```

Everything work's fine with my slider with just these scripts. However if I add the UI references I need for the touchscreen:

```
<script src="http://code.jquery.com/jquery.min.js"></script>
<script src="http://code.jquery.com/ui/1.8.17/jquery-ui.min.js"></script>

<script src="Scripts/jquery.ui.touch-punch.min.js"></script>
```

The slider functionality breaks, with the 'handle' just floating away from it's slider 'slot' when moved with a mouse, instead of sliding.

It seems that any three of these three lines will just cause the page to no longer function. I have even added the dependencies before the touch-punch library line like so

```
<script src="Scripts/jquery.ui.mouse.js"></script>
<script src="Scripts/jquery.ui.widget.js"></script>
<script src="Scripts/jquery.ui.touch-punch.min.js"></script>
```

But still no luck! I'm an experienced programmer, but a bit of a newb to JQuery, and would appreciate you help.

The test page is here:  http://www.xvams.com/xvams/jstest.aspx

Thanks

**Comment by Zenid** [comment]
Update: I found a solution:

Once I removed the two references to the jquery-ui.min.js and jquery-ui.min.js libraries, but left the touch-punch library on its own, it started working! 

I think the problem is that these two libraries will not work with the newer main, JQuery library "jqueryv1.10.2.js", presumably because the widget and mouse scripts are now included in the main JQuery library, and so are no longer needed in addition. Adding these other versions after the main library seems to be what's breaking it.

Would someone more knowledgeable on JQuery/touchscreen issues care to comment?
